### PR TITLE
tikv: Fix some words for TiKV API v2

### PR DESCRIPTION
**Pull request by pingyu**
Signed-off-by: pingyu <yuping@pingcap.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

- Add the description that TiDB/TxnKV/RawKV can be used at the same time, which is supported in v6.4.0.
- Fix the words that `tidb-server` can not be used for business.
- Change "Raw Key Value" to "RawKV" to unify the word for RawKV. As all other places using "RawKV" except here.
- Polish the warning by introducing TiKV-BR to solve the scenario when API v2 can not be enable/disable smoothly.

Rendered [here](https://github.com/pingyu/pingcap-docs-cn/blob/tikv-api-v2-fix/tikv-configuration-file.md#api-version-从-v610-版本开始引入).

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by pingyu** [comment]
@haojinming @zeminzhou PTAL, thanks~

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -414,20 +414,21 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
 
 ### `api-version` <span class="version-mark">从 v6.1.0 版本开始引入</span>
 
-+ TiKV 作为 Raw Key Value 存储数据时使用的存储格式与接口版本。
++ TiKV 作为 RawKV 存储数据时使用的存储格式与接口版本。
 + 可选值：
     + `1`：使用 API V1。不对客户端传入的数据进行编码，而是原样存储。在 v6.1.0 之前的版本，TiKV 都使用 API V1。
     + `2`：使用 API V2：
         + 数据采用 MVCC（Multi Version Concurrency Control）方式存储，其中时间戳由 tikv-server 从 PD 获取（即 TSO）。
+        + 数据根据使用方式划分范围，支持单一集群 TiDB、事务 KV、RawKV 应用共存。
```
Comment (haojinming) [inline tikv-configuration-file.md:422]: `事务 KV`  -> `TxnKV`， keep same tyle with `RawKV`?

**Review by haojinming — COMMENTED** [review]
REST LGTM

File: tikv-configuration-file.md (hunk 1 above)
Comment (haojinming) [reply tikv-configuration-file.md:422]: Got it.

**Review by haojinming — APPROVED** [review]
LGTM

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -414,20 +414,21 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
...
-        + 启用 API V2 后需要在集群中额外部署至少一个 tidb-server 以回收过期数据。注意该 tidb-server 不可提供读写服务。可以部署多个 tidb-server 以保证高可用。
+        + 启用 API V2 后需要在集群中额外部署至少一个 tidb-server 以回收过期数据。该 tidb-server 可同时提供数据库读写服务。可以部署多个 tidb-server 以保证高可用。
         + 需要客户端的支持。请参考对应客户端的 API V2 使用说明。
         + 从 v6.2.0 版本开始，你可以通过 [TiKV-CDC](https://github.com/tikv/migration/tree/main/cdc) 组件实现 RawKV 的 Change Data Capture (CDC)。
 + 默认值：1
 
 > **警告：**
 >
-> - **只能**在部署新的 TiKV 集群时将 `api-version` 的值设置为 `2`，**不能**在已有的 TiKV 集群中修改该配置项的值。由于 API V1 和 API V2 存储的数据格式不相同，如果在已有的 TiKV 集群中修改该配置项，会造成不同格式的数据存储在同一个集群，导致数据损坏。这种情况下，启动 TiKV 集群时会报 "unable to switch storage.api_version" 错误。
+> - 由于底层存储格式发生了重大变化，因此**仅当** TiKV 中只有 TiDB 数据时，可以平滑启用或关闭 API V2。其他情况下，需要新建集群，并使用 [TiKV-BR](https://github.com/tikv/migration/blob/main/br/README-cn.md) 进行数据迁移。
```
Comment (Oreoxmt) [inline tikv-configuration-file.md:431]:
```suggestion
> - 由于底层存储格式发生了重大变化，因此**仅当** TiKV 中只有 TiDB 数据时，可以平滑启用或关闭 API V2。其他情况下，需要新建集群，并使用 [TiKV Backup & Restore](https://github.com/tikv/migration/blob/main/br/README-cn.md) 工具进行数据迁移。
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: tikv-configuration-file.md (hunk 2 above)
Comment (pingyu) [reply tikv-configuration-file.md]: Fixed, PTAL~

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -414,20 +414,21 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
...
 + 可选值：
     + `1`：使用 API V1。不对客户端传入的数据进行编码，而是原样存储。在 v6.1.0 之前的版本，TiKV 都使用 API V1。
     + `2`：使用 API V2：
         + 数据采用 MVCC（Multi Version Concurrency Control）方式存储，其中时间戳由 tikv-server 从 PD 获取（即 TSO）。
+        + 数据根据使用方式划分范围，支持单一集群 TiDB、事务 KV、RawKV 应用共存。
         + 需要同时设置 `storage.enable-ttl = true`。由于 API V2 支持 TTL 特性，因此强制要求打开 `enable-ttl` 以避免这个参数出现歧义。
-        + 启用 API V2 后需要在集群中额外部署至少一个 tidb-server 以回收过期数据。注意该 tidb-server 不可提供读写服务。可以部署多个 tidb-server 以保证高可用。
+        + 启用 API V2 后需要在集群中额外部署至少一个 tidb-server 以回收过期数据。该 tidb-server 可同时提供数据库读写服务。可以部署多个 tidb-server 以保证高可用。
         + 需要客户端的支持。请参考对应客户端的 API V2 使用说明。
         + 从 v6.2.0 版本开始，你可以通过 [TiKV-CDC](https://github.com/tikv/migration/tree/main/cdc) 组件实现 RawKV 的 Change Data Capture (CDC)。
```
Comment (Oreoxmt) [inline tikv-configuration-file.md:426]:
```suggestion
        + 从 v6.2.0 版本开始，你可以通过 [RawKV CDC](https://github.com/tikv/migration/tree/main/cdc) 组件实现 RawKV 的 Change Data Capture (CDC)。
```

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Pull request merged**